packages/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -109,11 +110,30 @@ export class LockedGoldWrapper extends BaseWrapper<LockedGold> {
 
   // FIXME this.contract.methods.delegateRewards does not exist
   async delegateRewardsTx(account: string, delegate: string): Promise<CeloTransactionObject<void>> {
+    return this.delegateRoleTx(account, delegate, Roles.rewards)
+  }
+
+  async delegateVotingTx(account: string, delegate: string): Promise<CeloTransactionObject<void>> {
```

Comment:
Add [typedoc comments](https://typedoc.org/guides/doccomments/) here please.
Also, we can use `Address` instead of `string`.

I wonder if we should remove the `Tx` suffix, since we don't do that for any other `send` method